lib/tasks/load_csv.rake

```diff
@@ -1,10 +1,6 @@
 require 'csv'
 require './config/environment'
 
-desc "Print reminder about eating more fruit."
```

Comment:
testing a puts statement proved difficult. This was used as a setup test. not needed